src/types/Asset.ts

```diff
@@ -44,8 +44,22 @@ export enum SchneiderProperty {
 }
 
 export enum IothinkProperty {
-  POWER_ACTIVE = 'Puissance_active_totale',
   POWER_L1 = 'Puissance_active_de_phase_1',
   POWER_L2 = 'Puissance_active_de_phase_2',
   POWER_L3 = 'Puissance_active_de_phase_3',
+
+  // Generic
+  IO_POW_ACTIVE = 'io-pow-active',
```

Comment:
Why not creating an IoThinkAPIProperty enum?

Reply:
@LucasBrazi06 
Could you explain this a bit more? 
This is already an enum called IothinkProperty similar to the SchneiderProperty one.

Reply:
I will probably remove the values above the added ones, because they were used for the parking sud device and I think we will not need them anymore.

Reply:
An enum type that will contain constants (POWER_L1...)
And we may need to get the power per phase in the Optimizer from the Asset as discussed.

Reply:
POWER_L1 etc. are already in an enum type.
The problem is with the new devices for solar, battery and building we do not get the power per phase anymore.

Reply:
> POWER_L1 etc. are already in an enum type.
> The problem is with the new devices for solar, battery and building we do not get the power per phase anymore.

Weird, I haven't seen the enum in the first place.
That's ok then!
For the rest, if the power is not provided per phase and we have a 3 phases site area, divide it by 3.